src/Moq/Properties/Resources.resx

```diff
@@ -121,7 +121,7 @@
     <value>Expression {0} involves a field access, which is not supported. Use properties instead.</value>
   </data>
   <data name="TypeNotMockable" xml:space="preserve">
-    <value>Type to mock must be an interface, a delegate, or a non-sealed, non-static class.</value>
+    <value>Mocked type {0} must be an interface, a delegate, or a non-sealed, non-static class.</value>
```

Comment:
Since the type isn't actually mocked yet, _"mocked type"_ is less correct than the existing phrasing, _"type to mock"_. I'd suggest changing this to:

```suggestion
    <value>Type to mock ({0}) must be an interface, a delegate, or a non-sealed, non-static class.</value>
```